src/vcSettingsUI.cpp

```diff
@@ -762,80 +803,28 @@ void vcSettingsUI_VisualizationSettings(vcState *pProgramState, vcVisualizationS
     if (pVisualizationSettings->useCustomClassificationColours)
     {
       ImGui::SameLine();
-      if (ImGui::Button(udTempStr("%s##RestoreClassificationColors", vcString::Get("settingsRestoreDefaults"))))
-        memcpy(pVisualizationSettings->customClassificationColors, GeoverseClassificationColours, sizeof(pVisualizationSettings->customClassificationColors));
-
-      static const char *s_customClassifications[] =
-      {
-        "settingsVisClassUnclassified",
-        "settingsVisClassGround",
-        "settingsVisClassLowVegetation",
-        "settingsVisClassMediumVegetation",
-        "settingsVisClassHighVegetation",
-        "settingsVisClassBuilding",
-        "settingsVisClassLowPoint",
-        "settingsVisClassKeyPoint",
-        "settingsVisClassWater",
-        "settingsVisClassRail",
-        "settingsVisClassRoadSurface",
-        "settingsVisClassReserved",
-        "settingsVisClassWireGuard",
-        "settingsVisClassWireConductor",
-        "settingsVisClassTransmissionTower",
-        "settingsVisClassWireStructureConnector",
-        "settingsVisClassBridgeDeck",
-        "settingsVisClassHighNoise"
-      };
-
-      if (ImGui::Button(vcString::Get("settingsVisClassShowAll")))
-      {
-        for (size_t i = 0; i < udLengthOf(pVisualizationSettings->customClassificationToggles); i++)
-          pVisualizationSettings->customClassificationToggles[i] = true;
-      }
-
       ImGui::SameLine();
-      if (ImGui::Button(vcString::Get("settingsVisClassDisableAll")))
-      {
-        for (size_t i = 0; i < udLengthOf(pVisualizationSettings->customClassificationToggles); i++)
-          pVisualizationSettings->customClassificationToggles[i] = false;
-      }
+      if (ImGui::Button(udTempStr("%s##RestoreClassificationColors", vcString::Get("settingsRestoreDefaults"))))
+        memcpy(pProgramState->settings.visualization.customClassificationColors, GeoverseClassificationColours, sizeof(pProgramState->settings.visualization.customClassificationColors));
 
-      for (size_t i = 0; i < udLengthOf(s_customClassifications); i++)
-      {
-        ImGui::PushID((int)i);
-        ImGui::Checkbox("", &pVisualizationSettings->customClassificationToggles[i]);
-        ImGui::PopID();
-        ImGui::SameLine();
-        vcIGSW_ColorPickerU32(vcString::Get(s_customClassifications[i]), &pVisualizationSettings->customClassificationColors[i], ImGuiColorEditFlags_NoAlpha);
-      }
+      for (uint8_t i = 0; i < 19; ++i)
```

Comment:
This will break if the size of `customClassificationColors` changes. Maybe use 
```for (uint8_t i = 0; i < UDARRAYSIZE(settings.visualization.customClassificationColors); ++i)``` 
here as well.